first_timers/first_timers.py

```diff
@@ -1,7 +1,7 @@
 # -*- coding: utf-8 -*-
 from __future__ import print_function
 import re
-import warnings
+# import warnings
```

Comment:
Good catch. I think we should remove it in place of commenting it out

Reply:
I'll do that shortly